test/e2e/specs/site-editor/push-to-global-styles.spec.js

```diff
@@ -5,19 +5,18 @@ const { test, expect } = require( '@wordpress/e2e-test-utils-playwright' );
 
 test.describe( 'Push to Global Styles button', () => {
 	test.beforeAll( async ( { requestUtils } ) => {
-		await Promise.all( [
-			requestUtils.activateTheme( 'emptytheme' ),
-			requestUtils.deleteAllTemplates( 'wp_template' ),
-			requestUtils.deleteAllTemplates( 'wp_template_part' ),
-		] );
+		await requestUtils.activateTheme( 'emptytheme' );
```

Comment:
A small clean-up. The test doesn't create any templates or template parts, so there is no need to delete them.